git-hub

```diff
@@ -1513,8 +1516,12 @@ class PullCmd (IssueCmd):
                     force=args.force_push)
             infof("Creating pull request from branch {} to {}:{}",
                     remote_head, config.upstream, base)
-            pull = req.post(cls.url(), head=gh_head, base=base,
-                    title=title, body=body)
+            media_type = "application/vnd.github.v3+json"
```

Comment:
Sorry to be late with this comment ..
This `media_type` is already the default, so the ideal would be not to pass it at all. That implies something like 

```
if args.draft:
  media_type = ...
  pull = req.post(cls.url(), media_type, head=gh_head, base=base,
                    title=title, body=body, draft=args.draft)
else:
    pull = req.post(cls.url(), head=gh_head, base=base, title=title, body=body)
```

That is more verbose in that the `req.post` is repeated, but it is clearer too in that the default isn't overridden, and `args.draft` is only passed if there is in fact a draft.

Anyway, just a stylistic 2c.

Reply:
No problem. I'll amend it as you said it is clearer in a new PR. Thanks

Reply:
Addressed in #275